src/Helper/ChunkHelpers.php

```diff
@@ -68,4 +80,57 @@ public function storeChunk(StorageConfig $config, Range $range, UploadedFile $fi
 
         return Storage::disk($config->getDisk())->files($directory);
     }
+
+    /**
+     * List all chunks from a directory with the given name.
+     *
+     * @param \LaraCrafts\ChunkUploader\StorageConfig $config
+     * @param string $uuid
+     *
+     * @return array
+     */
+    public function chunks(StorageConfig $config, string $uuid): array
+    {
+        $directory = $config->getChunkDirectory() . '/' . $uuid;
```

Comment:
At first this could seem as a "useless" variable because it is only used once. However, the variable name it self contains information about the value it has.